docs/src/pages/docs/functions/combinators.md

```diff
@@ -309,6 +309,90 @@ with this library. It quite simply is just a function that when you pass it
 something, it returns that thing right back to you. So simple, I will leave it
 as an exercise to reason about why this is so powerful and important.
 
+#### psi
+
+`crocks/combinators/psi`
+
+```haskell
+psi ::  (b -> b -> c) -> (a -> b) -> a -> a -> c
+```
+
+`psi` is a function that can be considered the sister of [`converge`](#converge).
+Where [`converge`](#converge) takes one argument an maps it through two functions,
+merging the resulting values with an accumulating function, `psi` takes two
```

Comment:
```suggestion
merging the resulting values with an binary combination function, `psi` takes two
```

Reply:
`binary` is important. If there are other instances in docs where `accumulating` is preferred over `combining`, or if there is some other term, use that.

Reply:
So this is actually similiar to what i wrote originally, my preference is `binary` over `accumulating` however i changed this to match the phrasing of `converge` and have a common voice. 
Should we make a point of changing the docs for `converge`?

Reply:
I might also drop combination from your suggestion, i'm not sure it's neccessary

Reply:
sounds good to me. To my ear, binary is clearer than accumulating, so i'd prefer to see that jargon adopted globally @evilsoft ?

Reply:
Yeah i would agree 100%